pymongo/asynchronous/mongo_client.py

```diff
@@ -1040,6 +1041,20 @@ async def target() -> bool:
         self._kill_cursors_executor = executor
         self._opened = False
 
+    def _append_metadata(self, driver_info: DriverInfo) -> None:
```

Comment:
I think this needs to be public: "Specifies that drivers MUST provide an API to allow updating client metadata (name, version, platform) after initialization"

Reply:
oh good catch!